src/api-mocks/submissions.js

```diff
@@ -77,3 +77,20 @@ export const mockSubmissionProcessingStatusPendingGet = rest.get(
       })
     )
 );
+
+export const mockSubmissionProcessingStatusErrorGet = rest.get(
```

Comment:
When you ask Yoda, instead of GPT to name your things. :)

Consistent it is. Like it, I do.

Beats spelling out "Request" on every mock, as it's implied.